File: coffee_machine_python/beverage.py
# beverage.py
class Beverage:
    def __init__(self, name, water, beans, milk):
        self.name = name
        self.water = water
        self.beans = beans
        self.milk = milk

class Espresso(Beverage):
    def __init__(self):
        super().__init__('Espresso', 30, 20, 0)

class Latte(Beverage):
    def __init__(self):
        super().__init__('Latte', 200, 20, 150)

File: coffee_machine_java/src/Beverage.java
// src/Beverage.java
public abstract class Beverage {
    protected String name;
    protected int water;
    protected int beans;
    protected int milk;

    public Beverage(String name, int water, int beans, int milk) {
        this.name = name;
        this.water = water;
        this.beans = beans;
        this.milk = milk;
    }

    public String getName() {
        return name;
    }

    public int getWater() {
        return water;
    }

    public int getBeans() {
        return beans;
    }

    public int getMilk() {
        return milk;
    }
}
